extensions/notebook/src/jupyter/jupyterSessionManager.ts

```diff
@@ -333,6 +333,10 @@ export class JupyterSession implements nb.ISession {
 	private async setEnvironmentVars(skip: boolean = false): Promise<void> {
 		if (!skip && this.sessionImpl) {
 			let allCode: string = '';
+			// Ensure cwd matches notebook path (this follows Jupyter behavior)
+			if (this.path && path.dirname(this.path)) {
+				allCode += `%cd ${path.dirname(this.path)}${EOL}`;
```

Comment:
i think we need to escape the \ and space characters here, right?

Reply:
![image](https://user-images.githubusercontent.com/40371649/67916611-7a0c6200-fb53-11e9-8e26-398336abb1bc.png)

Jupyter takes care of '\'. Let me test spaces

Reply:
![image](https://user-images.githubusercontent.com/40371649/67916732-dec7bc80-fb53-11e9-8c3f-c5107856002f.png)

Looks ok there too

Reply:
great